Src/PerformanceCollector/Shared/Implementation/QuickPulse/QuickPulseServiceClient.cs

```diff
@@ -204,7 +205,7 @@ private void WriteSamples(IEnumerable<QuickPulseDataSample> samples, string inst
             this.serializerDataPointArray.WriteObject(stream, monitoringPoints.ToArray());
         }
 
-        private HttpWebResponse SendRequest(string httpVerb, string path, Action<Stream> onWriteBody)
+        private HttpWebResponse SendRequest(string httpVerb, string path, bool includeHeaders, Action<Stream> onWriteBody)
```

Comment:
Do we need optimize and not send this info for posts? Don't think it is a huge overhead. I'd rather simplify the logic here and send them always.